django/utils/http.py

```diff
@@ -290,6 +290,7 @@ def is_safe_url(url, host=None):
         url = url.strip()
     if not url:
         return False
+    url = force_text(url, errors='replace')
```

Comment:
Do you still want to remove this when dropping Python 2? If there isn't something like `six.PY2` will we remember? I guess `force_text()` still has some value in resolving lazy objects but that's not why it's used here. Just wondering...